Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -1252,6 +1248,35 @@ public static IMixedRealityBoundarySystem BoundarySystem
 
         private static bool logBoundarySystem = true;
 
+        private static IMixedRealityCameraSystem cameraSystem = null;
+
+        /// <summary>
+        /// The current Camera System registered with the Mixed Reality Toolkit.
+        /// </summary>
+        public static IMixedRealityCameraSystem CameraSystem
+        {
+            get
+            {
+                if (isApplicationQuitting)
+                {
+                    return null;
+                }
+
+                if (cameraSystem != null)
+                {
+                    return cameraSystem;
+                }
+
+                cameraSystem = Instance.GetService<IMixedRealityCameraSystem>(showLogs: logCameraSystem);
+                // If we found a valid system, then we turn logging back on for the next time we need to search.
```

Comment:
Interesting...what bug is this fixing, out of curiosity? I wonder if it could get turned off and then not back on.

Reply:
Not sure - again I'm just duping the registration method I see other core services use. I haven't delved into the logging system yet.